lyluatex.sty

```diff
@@ -120,9 +120,10 @@
     ['xml2ly'] = {'musicxml2ly'},
   })
 }
+\directlua{ly.make_list_file()}
 \directlua{
   if ly.get_option('cleantmp') then
-    luatexbase.add_to_callback('stop_run', ly.clean_tmp_dir, 'lyluatex cleantmp')
+    luatexbase.add_to_callback('stop_run', ly.ly.declare_package_options, 'lyluatex cleantmp')
```

Comment:
Why do you replace this callback ? At the end of compilation, there's no need to process options, while it's necessary do clean the tmp dir.
```suggestion
    luatexbase.add_to_callback('stop_run', ly.clean_tmp_dir, 'lyluatex cleantmp')
```

Reply:
Hm, I have no idea how that happened. Certainly not intentionally. Sorry